app/templates/components/user-list.hbs

```diff
@@ -1,17 +1,25 @@
 <table>
   <thead>
     <tr>
-      <th class='text-left' colspan=1>{{t 'user.firstName'}}</th>
-      <th class='text-left' colspan=1>{{t 'user.lastName'}}</th>
-      <th class='text-left' colspan=1>{{t 'general.school'}}</th>
+      <th class='text-left' colspan=1></th>
+      <th class='text-left' colspan=3>{{t 'user.fullName'}}</th>
+      <th class='text-left' colspan=2>Campus ID</th>
+      <th class='text-left' colspan=5>{{t 'general.email'}}</th>
+      <th class='text-left' colspan=2>{{t 'general.primarySchool'}}</th>
     </tr>
   </thead>
   <tbody>
     {{#each users as |user|}}
-      <tr class="{{if course.showRemoveConfirmation 'confirm-removal'}}">
-        <td class='text-left' colspan=1>{{user.firstName}}</td>
-        <td class='text-left' colspan=1>{{user.lastName}}</td>
-        <td class='text-left' colspan=1>{{user.school.title}}</td>
+      <tr class="{{unless user.enabled 'italicize'}}">
+        <td class='text-left user-status' colspan=1>
+          {{#unless user.enabled}}
+            {{fa-icon 'user-times' title=(unless user.enabled 'disabled')}}
```

Comment:
"disabled" needs to be translated.  Also probably doesn't need to be in an unless block since it is inside one that has the same function.

title=(t 'general.disabled')